conftest.py

```diff
@@ -67,11 +68,14 @@ def pytest_configure(config):
     """
     Configures pytest with the plotting and longrun command line options.
     """
-    if config.getoption("--plotting-on"):
-        tests.PLOTTING = True
-    else:
+    if not config.getoption("--plotting-on"):
         # We're not displaying plots so use a display-less backend
         mpl.use("Agg")
+        # Disable CAD viewer by mocking out QApplication.exec
+        from PySide2.QtWidgets import QApplication
```

Comment:
I dont know if the test error stems from here but you might need to change the backend to 'offscreen'